docs/containers/parallel-services.md

```diff
@@ -0,0 +1,46 @@
+---
+Area: containers
+ContentId: 
+PageTitle: Debug multiple services at the same time with Bridge to Kubernetes
+DateApproved: 05/24/2021
+MetaDescription: Learn how to use Bridge to Kubernetes to connect your development computer to a Kubernetes cluster and debug multiple services at the same time with local tunnel debugging, with Visual Studio Code.
+---
+# Debug multiple services at the same time
+
+Bridge to Kubernetes provides the ability to debug your Kubernetes services in a local environment, as described in [Use Bridge to Kubernetes](bridge-to-kubernetes.md).  With Bridge to Kubernetes, you redirect traffic to a locally running instance of a service and can debug using VS Code's debugger. However, in some scenarios, you want to work with more than one service and debug into them all at the same time. You can debug multiple services in parallel by following these steps.
```

Comment:
Nit: there are two spaces before `  With Bridge to Kubernetes`. Not sure if it would even be visible, so feel free to won't fix.

Reply:
I think that just gets handled, but I'll go ahead and fix it anyway.